OracleTSAM/samples/xe-db-svc/README.md

```diff
@@ -0,0 +1,42 @@
+Running Oracle TSAM Plus with oracle/database:11.2.0.2-xe image
+=============================
+You can run the TSAM container with the `oracle/database:11.2.0.2-xe` image provided on [GitHub](https://github.com/oracle/docker-images/tree/master/OracleDatabase).
+
+Due to the `oracle/database:11.2.0.2-xe` container creates new database and generates random `sys` user password on startup, a little "hacking" is needed here to make the `sys` password fixed.
+
+Basically this is done by overriding the default command `runOracle.sh` with this [modified one](cmd/runOracle.sh) in the `cmd` folder. The `docker-compose.yml` file to start the containers will look like below:
```

Comment:
Yeah, I think that that makes sense. So users have the best of both worlds, either pass on a password or just generate a random one.